mpas_analysis/shared/plot/plotting.py

```diff
@@ -44,35 +47,48 @@ def timeseries_analysis_plot(config, dsvalues, N, title, xlabel, ylabel, fileout
 
     Author: Xylar Asay-Davis
 
-    Last Modified: 02/02/2017
+    Last Modified: 02/11/2017
     """
-    plt.figure(figsize=figsize, dpi=dpi)
+    fig = plt.figure(figsize=figsize, dpi=dpi)
 
     for dsIndex in range(len(dsvalues)):
         dsvalue = dsvalues[dsIndex]
         if dsvalue is None:
             continue
         mean = pd.Series.rolling(dsvalue.to_pandas(), N, center=True).mean()
-        mean.plot(style=lineStyles[dsIndex], lw=lineWidths[dsIndex])
+        mean = xr.DataArray.from_series(mean)
+        plt.plot_date(mean['Time'], mean,
+                      fmt=lineStyles[dsIndex],
+                      xdate=True,
+                      ydate=False,
+                      linewidth=lineWidths[dsIndex])
+
+    ax = plt.gca()
+    # TODO: it would be good to change labels to include months if the plot is
+    # shorter than a couple of years
+    ax.xaxis.set_major_locator(mdates.YearLocator())
+    ax.xaxis.set_major_formatter(mdates.DateFormatter('%Y'))
+    ax.xaxis.set_minor_locator(mdates.MonthLocator())
+    fig.autofmt_xdate()
 
     if titleFontSize is None:
         titleFontSize = config.get('plot', 'titleFontSize')
 
-    axis_font = {'size':config.get('plot', 'axisFontSize')}
+    axis_font = {'size': config.get('plot', 'axisFontSize')}
     title_font = {'size': titleFontSize,
-                  'color':config.get('plot', 'titleFontColor'),
-                  'weight':config.get('plot', 'titleFontWeight')}
-    if (title != None):
+                  'color': config.get('plot', 'titleFontColor'),
+                  'weight': config.get('plot', 'titleFontWeight')}
+    if title is not None:
```

Comment:
We just had this discussion before to the other outcome, that is why I asked.  I agree that the `is not None` approach is better.  What we may want to add to the function signature is `title=None` as a default, but that is up to you.  It is the point of this PR because you are changing code, hence why I asked: `!=` changed to `is not`.  

This brings up a good point-- in the future we probably want to keep style / formatting changes to their own PRs for clarity.  We have been letting this slide for convenience but it is not best practices.  In other words, each PR should be more narrowly focused and only change the lines relevant to its scope.  This will also help greatly in the review process at the small cost of having more PRs to review, although syntax and formating-only PRs should be very easy to review, especially with the develop and master branch safety net.